src/Reflection/ReflectionClass.php

```diff
@@ -542,6 +534,35 @@ function (ReflectionClassConstant $constant) : string {
     }
 
     /**
+     * Get an array reflection object of the defined constants in this class.
+     *
+     * @return ReflectionClassConstant[]
+     */
+    public function getReflectionConstants() : array
+    {
+        return array_merge(
+            $this->getImmediateReflectionConstants(),
```

Comment:
I've improved the the already existed test.

I've also improved the implementation so the keys order in array should be same as in internal implementation: https://3v4l.org/lheaK